.buildkite/pipeline.yml

```diff
@@ -39,27 +23,34 @@ steps:
 
           echo "--- :rspec: Run Rspec"
           bundle exec rspec --profile 10 --format progress
-        env: *xcode_image
-        plugins: [*ci_toolkit]
+
+          echo "--- :fastlane: :buildkite: Test pipeline upload action"
+          bundle exec fastlane run buildkite_pipeline_upload pipeline_file:.buildkite/pipeline_for_upload_action_test.yml
```

Comment:
I considered having a dedicated step for this, and maybe we should, but at this point, before getting more feedback, I decided to leave in the same step as the tests.

What do you think?

A dedicated step might make things a bit clearer, but also seems like a lot (including extracting shared setup) for a simple action like this....